torch/csrc/api/src/nn/modules/embedding.cpp

```diff
@@ -11,27 +12,186 @@
 namespace torch {
 namespace nn {
 
-EmbeddingOptions::EmbeddingOptions(int64_t count, int64_t dimension)
-    : count_(count), dimension_(dimension) {}
+    EmbeddingImpl::EmbeddingImpl(const EmbeddingOptions& options_) : options(options_) {
```

Comment:
To address the clang-tidy error, we can add `// NOLINT(modernize-pass-by-value)` at the end of this line.